Tan_Heng_Yeow/progress.md

```diff
@@ -22,20 +22,42 @@ Overall Direction:
 ---
 
 Week 1 Plan:
-- [ ] Finish up the remaining work for this [PR](https://github.com/TEAMMATES/teammates/pull/7568)
+- [X] Finish up the remaining work for this [PR](https://github.com/TEAMMATES/teammates/pull/7568)
 - [ ] Write tests for features in the spreadsheet interface
 
+Works:
+- 1st review ready at https://github.com/TEAMMATES/teammates/pull/8833#issuecomment-389790326
+- Highlighted some thoughts on plan ahead at https://github.com/TEAMMATES/teammates/issues/8832#issuecomment-389788642
+
+Retrospect on Week 1:
+- Understanding what have been implemented so far
```

Comment:
Retrospect is not a list of what you have done. It is more of a reflection on what you have done well, what could be improved, what are the difficulties faced, etc.